Twig/TokenParser/ExtendsAdmingeneratedTokenParser.php

```diff
@@ -22,7 +22,7 @@ public function parse(\Twig_Token $token)
 
         $path = "Admingenerated/$bundle/Resources/views/$folder/$file";
 
-        $value = $this->parser->getExpressionParser()->parseExpression();
+        $this->parser->getExpressionParser()->parseExpression();
```

Comment:
@sescandell This change was [discussed here](https://github.com/symfony2admingenerator/AdmingeneratorGeneratorBundle/issues/704), so we can't remove this line, but since `$value` is not used anywhere, we don't have to assign it.